### DISCUSSION TITLE
제페토 캐릭터의 중력값 조절법이 있을까요?

### DISCUSSION THREAD

**Discussion by CMSFM** [naverz/zepeto-studio-kor#999] [5. Q&A]
현재 월드 버전은 1.5.1 , 스크립트버전은 1.4.1입니다. 

제페토 플레이어를 무중력 상태로 조종하거나 낮은 중력으로 달과 같은 맵을 만들기 위해서 중력 값을 조절해보고 싶은데요

이전에는 분명 Hierarchy의 ZepetoPlayers에 Gravity를 조정하는 값이 있었던거 같은데 없어졌더군요.

그래서 코드로 조정하려고 하니, 
        ZepetoPlayers.gravity = 0;
라는 값이 있어 0으로 바꾸고 테스트를 해봤는데, 극적인 변화는 없었습니다.

혹시 해당 부분에 관해서 적절한 아이디어가 있으신 분들이 있으실까요?

분명 전에는 중력값 조절이 있었던거 같은데...왜 없어졌을까요.

심지어 이거 RigidBody도 아니고 CharacterController라서 중력값을 코드로 조절하시고 있으실텐데, 만약 없애신거라면 조속히 되돌려 주셨으면 합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 사용하신 ZepetoPlayers.gravity = 0; 코드의 경우 캐릭터 컨트롤러 모션 V1을 사용하는 캐릭터 전체에 일괄적으로 적용되는 중력값입니다. (개별 캐릭터 설정 불가)

월드 버전 1.1.6 버전에 모션 V2가 업데이트 되면서 V2 캐릭터에만 적용되는 중력값 제어 변수가 추가 되었습니다.
```typescript
character.MotionV2.Gravity = (number);
```  
를 사용하면 지금도 가능합니다.

다만 기존에 존재하던 중력 변수 (ZepetoPlayers.gravity) 와
모션 V2부터 새롭게 추가한 중력 변수 (ZepetoCharacter.MotionV2.Gravity) 의 작용방식이 달라, ZepetoPlayers 인스펙터에 노출시킬 경우 오히려 혼란의 여지가 있다고 판단되어 월드 버전 1.3.0 에서부터 인스펙터에서 제거하였습니다.

알려드린 코드로 사용 부탁 드립니다.
감사합니다.

**Reply by CMSFM** [reply]
감사합니다....! 덕분에 막힌 부분이 좀 풀린거 같아요!
